problem_builder/public/js/problem_builder.js

```diff
@@ -0,0 +1,40 @@
+window.ProblemBuilder = {
+
+    transformClarifications: function(element) {
+        element = $(element);
```

Comment:
Ah, ok. I figured you probably had a good reason for it, but it doesn't hurt to ask :)